### ISSUE TITLE
CodeChunk: Accessibility check failures

### ISSUE THREAD

**Issue by nokome** [stencila/designa#37]
Over in Encoda we are doing AXE and WCAG2 accessibility checks on generated HTML. There are several failures for `CodeChunks` that we are currently skipping (not some of these are double ups):

```
    Failed: "html > body > main > article > stencila-code-chunk:nth-child(8) > div > stencila-code-editor > div > menu > select: label: Form elements must have labels (https://dequeuniversity.com/rules/axe/3.5/label?application=axeAPI)"

    Failed: "html > body > main > article > stencila-code-chunk:nth-child(11) > div > stencila-code-editor > div > menu > select: label: Form elements must have labels (https://dequeuniversity.com/rules/axe/3.5/label?application=axeAPI)"

    Failed: "html > body > main > article > stencila-code-chunk:nth-child(8) > stencila-action-menu > nav: landmark-unique: Ensures landmarks are unique (https://dequeuniversity.com/rules/axe/3.5/landmark-unique?application=axeAPI)"

    Failed: "html > body > main > article > stencila-code-chunk:nth-child(11) > div > stencila-code-editor > div > div > div:nth-child(2) > div: scrollable-region-focusable: Ensure that scrollable region has keyboard access (https://dequeuniversity.com/rules/axe/3.5/scrollable-region-focusable?application=axeAPI)"

    Failed: "html > body > main > article > stencila-code-chunk:nth-child(8) > div > stencila-code-editor > div > menu > select: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name: This select element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined."

    Failed: "html > body > main > article > stencila-code-chunk:nth-child(8) > div > stencila-code-editor > div > menu > select: WCAG2AA.Principle1.Guideline1_3.1_3_1.F68: This form field should be labelled in some way. Use the label element (either with a \"for\" attribute or wrapped around the form field), or \"title\", \"aria-label\" or \"aria-labelledby\" attributes as appropriate."

    Failed: "html > body > main > article > stencila-code-chunk:nth-child(11) > div > stencila-code-editor > div > menu > select: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name: This select element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined."

    Failed: "html > body > main > article > stencila-code-chunk:nth-child(11) > div > stencila-code-editor > div > menu > select: WCAG2AA.Principle1.Guideline1_3.1_3_1.F68: This form field should be labelled in some way. Use the label element (either with a \"for\" attribute or wrapped around the form field), or \"title\", \"aria-label\" or \"aria-labelledby\" attributes as appropriate."

```
